e2e/utils/index.ts

```diff
@@ -91,6 +91,17 @@ export function readProjectConfig(projectName: string): ProjectConfiguration {
   return readJson(path);
 }
 
+export function createNonNxProjectDirectory(name = uniq('proj')) {
+  projName = name;
+  ensureDirSync(tmpProjPath());
+  createFile(
+    'package.json',
+    JSON.stringify({
+      name: projName,
```

Comment:
why not just `JSON.stringify({ name })`?

Reply:
You're right, that works. We have to assign to projName still in line 96, so that the readFile functions and such look in the correct directory. I went ahead and updated the JSON stringify though. 

Thanks Miro.